crates/typst-library/src/foundations/array.rs

```diff
@@ -815,6 +815,17 @@ impl Array {
     ///
     /// Returns an error if two values could not be compared or if the key
     /// function (if given) yields an error.
+    /// 
+    /// The key function can return a tuple to sort by multiple keys.
+    ///
+    /// ```example
+    /// #let array = (
+    ///   (a: "foo", b: "baz"),
+    ///   (a: "fizz", b: "buzz"),
+    ///   (a: "foo", b: "bar"),
+    /// )
+    /// #array.sorted(key: it => (it.a, it.b))
+    /// ```
```

Comment:
i was thinking about that but decided against it to also show that the sorting is alphabetically ascending.

Reply:
If the point is that arrays can be sorted, is showing off string sorting really relevant? Maybe it is too much at once.

Reply:
I'll also say that this should explicitly mention that this is a way to sort according to multiple criteria at once, in case of equality between some criteria. (I know that "lexicographic order", by itself, explains this; this'd be more of an additional short explanation for the average user.)

Reply:
I'm stealing this comment and update the docs accordingly, since you found a way better description than me.